lib/monitoring/addon/components/nodes-live/component.js

```diff
@@ -168,37 +145,14 @@ export default NodesReservation.extend(Metrics, {
     })
   }),
 
-  memoryLive: computed('graphs.[]', function() {
-    if (!get(this, 'graphs')) {
-      return
-    }
-
-    const memoryGraph = (get(this, 'graphs')).filter((g) => g.graph.title === MOMORY_GRAPH_TITLE)[0]
-
-    if (!memoryGraph || memoryGraph === undefined) {
-      return
-    }
-
-    const { series = [] } = memoryGraph
-    const { nodes = [] } = this
-    let out = 0
-
-    const filtered = series.filter((s) => {
-      return nodes.filter((n) => n.nodeName === s.name).length > 0
-    })
-
-    filtered.map((s) => out += (s.points[1] && s.points[1][0] || 0))
-
-    return Math.round((out / filtered.length) * 100)
-  }),
-
   cpuLiveTitle: computed('nodeCapcity', 'graphs.[]', 'cluster.capacity', 'intl.locale', function() {
-    const { cpuLive, nodes = [] } = this
+    const { graphs, nodes = [] } = this
 
-    if (!cpuLive) {
+    if (!graphs) {
       return 'loading'
     }
-    const { nodeCapcity = {}, graphs = [] } = this
+
+    const { nodeCapcity = {} } = this
```

Comment:
Is there any reason this can't be included on the destructuring assignment on line 149?